lib/transactions.js

```diff
@@ -99,6 +108,25 @@ TxController.prototype.transformTransaction = function(transaction, options, cal
 
   transformed.txlock = transaction.txlock;
 
+  if (transaction.proRegTx !== undefined) {
```

Comment:
Where `transaction.proRegTx` field is coming from?

Reply:
I've checked the output of the `getDetailedTransaction`, the field doesn't seem to be there

Reply:
You have to check a TX that registers a MN. For example `c751da008151a1daa470bdbae109287847fcfe0a76cba5f4c507c5c7ecc9b799`. The field is coming directly from dashd's `getrawtransaction <hash> 1` call.

Reply:
If you are looking for the specific line @antouhou it is here:
https://github.com/dashpay/dash/blob/develop/src/rpc/rawtransaction.cpp#L140

Reply:
@codablock @Cofresi Ah, I see, I was looking at `dashcore-node`'s master, and this is in the `develop`